dataprocspawner/spawner.py

```diff
@@ -1343,6 +1350,7 @@ def _build_cluster_config(self, cluster_data=None):
     # but must be set in case there is no form.
     cluster_data = cluster_data or {}
     cluster_zone = self.zone
+    prop_perso = 'dataproc:dataproc.personal-auth.user'
```

Comment:
Very minor nit: this variable name seems too abbreviated. Can we make it more descriptive like "personal_auth_property"?

Reply:
done